.github/workflows/validate-frontend.yaml

```diff
@@ -0,0 +1,24 @@
+name: validate
+on: pull_request
+
+jobs:
+  frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+      - name: Use Node v12.22
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.22
+      - name: Setup Cache & Install Dependencies
+        uses: bahmutov/npm-install@v1
+        with:
+          install-command: yarn --frozen-lockfile
+          working-directory: ./frontend
```

Comment:
@prashant-shahi can we define the working dir at the job level so that we can avoid this redundant definition. Tried various approaches nothing worked

Reply:
not sure if we can define working dir at job level but let @prashant-shahi confirm

Reply:
You can use "defaults" for that.

https://docs.github.com/en/actions/using-jobs/setting-default-values-for-jobs